src/SDKs/Monitor/Management.Monitor/generate.cmd

```diff
@@ -1,8 +1,54 @@
+:: 
+:: Microsoft Azure SDK for Net - Generate library code 
+:: Copyright (C) Microsoft Corporation. All Rights Reserved. 
+:: 
+ 
+@echo off 
+
+goto newCode
+
+if  "%2" == "" (
+   set specFile1="https://raw.githubusercontent.com/Azure/azure-rest-api-specs/5288e653adfe3dd6ce235a790008a8215eedb918/monitor/compositeMonitorClient.json"
+) else (
+   set specFile1="%2"
+)
+
+if  "%3" == "" (
```

Comment:
@gucalder you do not need all this code. Any reason you are updating this file.
Simply executing this file will create a _meta file that is what the new way to generate the code.
All this change has already been done for you, you only need to execute generate.cmd the way it is.
Please revert this change.
Once you execute generate.cmd, it will create a .txt file under _metadata directory like below
https://github.com/Azure/azure-sdk-for-net/tree/psSdkJson6/src/SDKs/_metadata